crd-convert/src/main/java/io/strimzi/kafka/crd/convert/server/ConversionReview.java

```diff
@@ -0,0 +1,80 @@
+/*
```

Comment:
I guess we should keep the `server` package out of this PR for now. It makes it easier to review and we don't need it right now. We can always add it later.

Reply:
@alesj  @tombentley can you explain me what this server is about, please?